active_directory/datadog_checks/active_directory/data/conf.yaml.example

```diff
@@ -1,46 +1,75 @@
 init_config:
 
 instances:
-  # "." means the current host
+
+  ## @param host - string - required
+  ## host to connect to.
+  ## "." means the current host 
+  # 
   - host: .
-  # username: my_username
-  # password: my_password
+
+  ## @param username - string - optional
+  ## XXXXXXXXXXX
   #
-  #   The additional metrics is a list of additional counters to collect.  The
-  #   list is formatted as follows:
-  #   ['<counterset name>', <counter instance name>, '<counter name>', <metric name>, <metric type>]
+  #  username: <USER_NAME>
+
+  ## @param password - string optional
+  ## XXXXXXXXXXX
   #
-  #   <counterset name>  is the name of the PDH Counter Set (the name of the counter)
-  #   <counter instance name> is the specific counter instance to collect, for example 
-  #           "Default Web Site".  Specify 'none' For all instances of the counter.
-  #   <counter name> is the individual counter to report
-  #   <metric name> is the name you want to show up in Datadog
-  #   <metric type> is from the standard choices for all agent checks, such as gauge,
-  #       rate, histogram or counter
-  #   
-  #   additional_metrics:
-  #     - ['NTDS', none, 'DS % Writes from LDAP', active_directory.ds.writes_from_ldap, gauge]
-  # 
-  # `counter_data_types` allows the precision with which counters are queried on a per metric basis
-  # counter_data_types:
-  #   - <metric_name>,<data_type>
-  #   - active_directory.dra.inbound.bytes.total,int
-  #   - active_directory.ldap.bind_time,float
+  #  password: <PASSWORD>
+
+  ## @param additional_metrics - array - optional
+  ## The additional metrics is a list of additional counters to collect.  The
+  ## list is formatted as follows:
```

Comment:
`The list` here is ambiguous. Something like `Each item is a list of strings, formatted as follows: ...` sounds clearer IMO